bindings/python/Makefile.am

```diff
@@ -0,0 +1,32 @@
+SUBDIRS = . tests
+
+AM_CXXFLAGS = -Wall
+
+BUILT_SOURCES = $(srcdir)/pire_pywrap.cpp
+SWIG_SOURCES = \
+	$(srcdir)/common.i \
+	$(srcdir)/fsm.i \
+	$(srcdir)/lexer.i \
+	$(srcdir)/scanners.i \
+	$(srcdir)/pire.swg
+
+TARGET_PYMODULE = $(srcdir)/pire.py
+
+MAINTAINERCLEANFILES = $(BUILT_SOURCES) $(TARGET_PYMODULE)
+
+pkgpython_PYTHON =
+pkgpyexec_LTLIBRARIES =
+
+if ENABLE_PYTHON
+
+pkgpython_PYTHON += $(TARGET_PYMODULE)
+pkgpyexec_LTLIBRARIES += _pire.la
```

Comment:
https://www.python.org/dev/peps/pep-0008/#package-and-module-names

When an extension module written in C or C++ has an accompanying Python module that provides a higher level (e.g. more object oriented) interface, the C/C++ module has a leading underscore (e.g. _socket ).